### PR TITLE
[#76032364] Document templates flow

### PR DESCRIPTION
**Pull request by hivtushok**
This PR documents a workflow for templates & auto-generated smart-contract tables.

### CODE REVIEW

**Comment by hivtushok** [comment]
Let me know if something is still not clear or can be reworded in a better way.

File: README.md [hunk 1]
```diff
@@ -13,6 +13,27 @@ The live version of the documentation is synced with [live-docs repository](http
...
+	2. Run `render:cns-contracts`
+
+###### Auto-generated smart-contract address tables:
+
+Contract addresses are pulled from [network-config.json](https://github.com/unstoppabledomains/dot-crypto/blob/master/src/network-config/network-config.json "network-config.json") contained in [dot-crypto](https://github.com/unstoppabledomains/dot-crypto).
+
+- To fetch new config run `render:cns-contracts`. 
+- If you need to add a new Contract:
+
+	1. Create temlate for the target doc with the pattern of ``<target-doc-name>-temaplte.md`` and put it in a */template* folder.
```
Comment (mvlabat) [inline README.md:30]:
```suggestion
	1. Create temlate for the target doc with the pattern of ``<target-doc-name>-template.md`` and put it in a */template* folder.
```

Btw, I'm not sure what `*/template*` folder is. I can't find any folder with such a pattern.

File: templates/contracts/contract-table-template.ejs [hunk 2]
```diff
@@ -1,3 +1,4 @@
+[comment]: <> (This is an auto-generated table with CNS smart-contract addresses)
```
Comment (mvlabat) [inline templates/contracts/contract-table-template.ejs:1]: This is nice. Could we also add a similar comment at the top of `src/domain-registry-essentials/cns-smart-contracts.md`, to point developers to `templates/cns-smart-contracts-template.md`?

File: README.md [hunk 3]
```diff
@@ -13,6 +13,27 @@ The live version of the documentation is synced with [live-docs repository](http
...
+	2. Inside of a template insert `#include "templates/contracts/<YourContractName>.md"` where you want to add your table.
```
Comment (mvlabat) [inline README.md:31]:
```suggestion
	2. Inside of the template insert `#include "templates/contracts/<YourContractName>.md"` where you want to add your table.
```

File: README.md [hunk 4]
```diff
@@ -13,6 +13,27 @@ The live version of the documentation is synced with [live-docs repository](http
...
+	3. Generate tables & render the template using the following commands:
+		- `render:cns-contracts` for *cns-smart-contracts.md*
+		- Or define your own for other files.
```
Comment (mvlabat) [inline README.md:34]: I'd probably not mention that and focus this readme just on the contract list to make it less confusing.

File: README.md (hunk 1 above)
Comment (hivtushok) [reply README.md]: It should be a '/templates' folder, thanks for noticing that.

File: templates/contracts/contract-table-template.ejs (hunk 2 above)
Comment (hivtushok) [reply templates/contracts/contract-table-template.ejs]: ✅

File: README.md (hunk 3 above)
Comment (hivtushok) [reply README.md]: ✅

File: README.md (hunk 4 above)
Comment (hivtushok) [reply README.md]: Removed parts that are out of scope.

**Review by DeRain — APPROVED** [review]

**Review by merenkoff — APPROVED** [review]

**Review by mvlabat — APPROVED** [review]

**Pull request merged**